frontends/web/src/components/copy/Copy.css

```diff
@@ -1,49 +1,52 @@
 .button {
```

Comment:
why did you change the style and position of the copy button? tbh I liked the previous version much better, it looked more polished.

Reply:
I'm still not certain that one looks better over another, but since other inputs such as the address field in send has the qr code inside the field, thought it may be good to have the same style across the app.